src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/DisposeAnalysis/DisposeAnalysisHelper.cs

```diff
@@ -37,10 +38,10 @@ internal sealed class DisposeAnalysisHelper
             OperationKind.Invocation);
 
         private readonly WellKnownTypeProvider _wellKnownTypeProvider;
-        private readonly ImmutableHashSet<INamedTypeSymbol> _disposeOwnershipTransferLikelyTypes;
-        private ConcurrentDictionary<INamedTypeSymbol, ImmutableHashSet<IFieldSymbol>> _lazyDisposableFieldsMap;
-        public INamedTypeSymbol IDisposable { get; }
-        public INamedTypeSymbol Task { get; }
+        private readonly ImmutableHashSet<INamedTypeSymbol>? _disposeOwnershipTransferLikelyTypes;
```

Comment:
💡 You can avoid making this nullable by initializing it to `ImmutableHashSet<INamedTypeSymbol>.Empty` on the constructor path that previously left it uninitialized.